### PR TITLE
update ci for no action on branching

### PR DESCRIPTION
**Pull request by chrbeckm**

### CODE REVIEW

File: .github/workflows/remove_template_files.yml [hunk 1]
```diff
@@ -1,26 +1,30 @@
...
 
 jobs:
+
+  # Run a cleaning process only if the workflow is triggered by the non-"template_test" repository.
   template-cleanup:
-    name: template-cleanup
+    name: Template Cleanup
     runs-on: ubuntu-latest
-    
+    if: github.event.repository.name != 'template_test'
```
Comment (aknierim) [inline .github/workflows/remove_template_files.yml:15]: Wouldn't that still be true for this repository and thus remove files?

**Review by aknierim — APPROVED** [review]
LGTM

**Comment by chrbeckm** [comment]
I tested it in a separate repository: https://github.com/chrbeckm/template_test

**Comment by aknierim** [comment]
Does the CI fail because of `TEXLIVE_VERSION: 2023` on line 12?

**Pull request merged**